packages/holocron/API.md

```diff
@@ -220,12 +286,22 @@ Adds a Holocron module to the registry
 | `moduleName` | `String` | `true` | The name of your Holocron module |
 | `module` | `Function` | `true` | The Holocron module itself (a React component) |
 
+#### `addHigherOrderComponent`
```

Comment:
This isn't in the public API right???

Reply:
I saw the other module registry methods exported in the index and thought to follow convention. I will remove this.

Reply:
Fixed in 3a805b5